bokehjs/src/lib/models/transforms/transform.ts

```diff
@@ -8,16 +8,16 @@ export namespace Transform {
   export type Props = Model.Props
 }
 
-export interface Transform<To = number> extends Transform.Attrs {}
+export interface Transform<From=number, To = number> extends Transform.Attrs {}
```

Comment:
why the inconsistent spacing around `=`?

Reply:
No good reason.